### PR TITLE
Command Tag Implementation

### PR DESCRIPTION
**Pull request by TreemanKing**

### CODE REVIEW

**Comment by TreemanKing** [comment]
Things to add before merge: 
- [ ] Command Level permission nodes
- Check if player is op or has * before temporarily giving it

File: core/src/main/java/com/sekwah/advancedportals/core/tags/activation/CommandTag.java [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    }
+
+    @Override
+    public void destroyed(TagTarget target, PlayerContainer player, String[] argData) {
+
+    }
+
+    public enum CommandLevel{
+        OP,
+        STAR,
```
Comment (sekwah41) [inline core/src/main/java/com/sekwah/advancedportals/core/tags/activation/CommandTag.java:127]: I think star should be referred to permission not star

**Review by sekwah41 — CHANGES_REQUESTED** [review]

File: core/src/main/java/com/sekwah/advancedportals/core/tags/activation/CommandTag.java (hunk 1 above)
Comment (TreemanKing) [reply core/src/main/java/com/sekwah/advancedportals/core/tags/activation/CommandTag.java:127]: > I think star should be referred to permission not star
```suggestion
        PERMISSION_WILDCARD,
```
Maybe this instead?

Comment (sekwah41) [reply core/src/main/java/com/sekwah/advancedportals/core/tags/activation/CommandTag.java:127]: yep :D

**Comment by TreemanKing** [comment]
This needs to be reviewed ASAP as Velocity/Bungee tags are reliant on this.

**Comment by sekwah41** [comment]
pre-commit.ci run

**Comment by TreemanKing** [comment]
If everything here is all good in RE: to command tags, probably best to merge and then add further commits via PRs

**Comment by sekwah41** [comment]
Merging into an off-branch and will finish. Currently it behaves different to the old tags though is good solid base for 95% of it all :)

**Pull request merged**